docs/snippets/html/button-story.js.mdx

```diff
@@ -0,0 +1,13 @@
+```js
+import { createButton } from './Button';
+
+export default {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/html/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'Button',
+};
+
+export const Primary = () => createButton({ primary: true, label: 'Button' });
```

Comment:
This file looks like the generated `Button.stories.js` of `npx sb init` for an HTML project, which is an excellent starting point for this snippet except for one detail:

It's not obvious what `createButton` ([source](https://github.com/storybookjs/storybook/blob/next/lib/cli/src/frameworks/html/js/Button.js)) does. I think it would be more clear if you replaced the call to `createButton` with its output, to avoid the abstraction. What do you think?

Reply:
This is good point.
I've update snippets with `createButton` output but change it to make it shorter.
Would you mind to review it again?